extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
There are some standards, but mostly for what is the HTTP cache we serve, not the Memcached cache times.